### DISCUSSION TITLE
Removing SkiaLayout's Children with Clear doesn't work

### DISCUSSION THREAD

**Discussion by arifu-dev** [taublast/DrawnUi#108] [Q&A] [answered]
`layout.Children.Clear();` does not remove any child views, removing them individually with `layout.Children.RemoveAt(i);` works as expected.

**Comment by taublast** [comment, marked as answer]
Okay so the correct way is to use `layout.ClearChildren()`. 
Also you are not required to call this on UI thread at all, contrary to usual MAUI.

**Comment by taublast** [comment]
To tell more different custom layouts can have their own logic and implementations of handling children so they can override the `public virtual void ClearChildren` to handle that.

**Comment by arifu-dev** [comment]
Yep, `layout.ClearChildren()` works, not sure how I missed it.
